plugins/action_files_cleanup/src/databaseMethods.ts

```diff
@@ -9,6 +9,10 @@ const databaseMethods = (DBConnect: any) => ({
     FROM file
     WHERE application_serial = $1
     AND submitted = false
+    AND application_response_id IS NOT NULL
+    AND is_output_doc = false
+    AND is_internal_reference_doc = false
+    AND is_external_reference_doc = false
```

Comment:
I think it would have been enough to just add the `AND application_response_id IS NOT NULL` line, but no harm in being even more rigorous.